### PR TITLE
feat(web): added edit profile modal

### PR DESCRIPTION
**Pull request by Simplyalex99**
Added Edit Modal component in Components directory.

Also added root div to render modal in _document.tsx file

Also added state in user/[username]/index.tsx
to detect when modal is clicked or not. Added change from "follow" to "edit profile" and conditional to render modal depending on the state value.

Changed edit.tsx by removing navbar.

Requires css to be converted to tailwindcss in EditModal.tsx

Note: ormconfig.tsx file is not included in push request. In addition, thoughts on implementing custom Modal instead for both edit profile option and send message? Thanks!

### CODE REVIEW

File: packages/web/src/components/EditModal.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  backgroundColor: '#232936',
+  borderRadius: '50px',
+  outline: 'none',
+  zIndex: 1000,
+  display: 'inline-block',
+  transform: 'translate(-100%, -10%)',
+
+
+
+};
```
Comment (HenryLeC) [inline packages/web/src/components/EditModal.tsx]: Can you change this to tailwind before merge please

**Review by HenryLeC — CHANGES_REQUESTED** [review]

**Comment by Simplyalex99** [comment]
Hello,  i forgot to add one comment about the typeof window==undefined in EditModal.tsx. This was to prevent nextjs from crashing sometimes because it  might check server side first and DOM is undefined at that point.

File: packages/web/src/components/EditModal.tsx (hunk 1 above)
Comment (Simplyalex99) [reply packages/web/src/components/EditModal.tsx]: Hello, I will try , Acronie was planning on doing the tailwind changes but il give it a go.

File: packages/api/schema.gql [hunk 2]
```diff
@@ -64,6 +64,7 @@ type Mutation {
   editPost(data: EditPostInput!, postId: String!): Boolean!
   giveBadge(badgeName: String!, username: String!): Boolean!
   joinResort(resortId: String!): Boolean
+  likeDislike(dislike: Boolean!, like: Boolean!, postId: String!): Boolean!
   makeAdmin(roles: [Role!]!, user: String!): Boolean!
   makeBadge(data: MakeBadgeInput!): Boolean!
   markNotificationAsRead(notificationId: String!): Boolean!
```
Comment (BronzW) [inline packages/api/schema.gql:70]: why is this being changed

Comment (BronzW) [reply packages/api/schema.gql:70]: wait nvm

**Review by BronzW — APPROVED** [review]

**Pull request closed without merging**